python/beeai_framework/context.py

```diff
@@ -80,8 +86,6 @@ def _set_context(self, context: dict) -> None:
 
 
 class RunContext(RunInstance):
-    storage: ContextVar[Self] = ContextVar("storage", default=None)
```

Comment:
Why did you remove it? It seems more straightforward to have it there as you are using it in the class.

Reply:
mypy was upset we were referencing an instance var from a static method. Then when I looked into some fixes for that I found that ContextVar are designed be global vars not instance vars